llama-index-packs/llama-index-packs-secgpt/examples/SecGPT.ipynb

```diff
@@ -0,0 +1,741 @@
+{
```

Comment:
<p>I'm a bit confused as to how these tools actually provide the fare price of both ride-sharing apps? Is this purely for illustration? In other words, is this notebook not actually functional?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/13127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
You’re right! We developed these tools for attack illustration purposes. Developers can follow our examples to include their own real tools in SecGPT.